lambda/finalizer.js

```diff
@@ -88,3 +98,25 @@ const findFastest = (stats) => {
     // just return the first one
     return stats[0];
 };
+
+const findBalanced = (stats, w) => {
```

Comment:
Please use `weight` instead of `w`. It's just more readabale :)

Reply:
Fixed